### PR TITLE
Remove mobile view resolution and the mark_desktop_only skip reason in tests

### LINKED ISSUE #97: Remove mobile view tests for now
Right now we are skipping most, if not all, of the mobile view tests. We should enable these and make sure they're working still.

@AlexandraMoga would mobile view tests help softvision?

### PR DESCRIPTION
**Pull request by AlexandraMoga**
Fix #97

### CODE REVIEW

File: test_search.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import time
 import pytest
-
```
Comment (jrbenny35) [inline test_search.py:3]: These spaces should still be here. See, https://www.python.org/dev/peps/pep-0008/#imports

File: test_search.py [hunk 2]
```diff
@@ -1,11 +1,9 @@
...
 from selenium.common.exceptions import NoSuchElementException
 from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support.select import Select
-
```
Comment (jrbenny35) [inline test_search.py:8]: These spaces should still be here. See, https://www.python.org/dev/peps/pep-0008/#imports

File: conftest.py [hunk 3]
```diff
@@ -1,11 +1,9 @@
 import os
-
```
Comment (jrbenny35) [inline conftest.py:2]: These spaces should still be here. See, https://www.python.org/dev/peps/pep-0008/#imports

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Looks good, just some styling comments

**Comment by AlexandraMoga** [comment]
While updating the the files included in the initial patch, I've noticed that a few other files needed the same reorganization, so I've updated those too, although they were not in the initial scope of this patch.

**Comment by jrbenny35** [comment]
> 
> 
> While updating the the files included in the initial patch, I've noticed that a few other files needed the same reorganization, so I've updated those too, although they were not in the initial scope of this patch.

This time no biggie, but lets file an issue for things like this next time 😃

**Review by jrbenny35 — APPROVED** [review]

**Pull request merged**